src/handler.rs

```diff
@@ -177,12 +181,84 @@ struct ObsBuildInfo {
     enabled_repos: HashMap<RepoArch, CommitBuildInfo>,
 }
 
+#[derive(Clone, Debug)]
```

Comment:
I think the remaining part of this file deserves a little explanation 🙂
Probably in comments, likely also in the commit message.

Reply:
Is `StringFile` used only in tests?